localassetmanager.js

```diff
@@ -1,7 +1,7 @@
-﻿import filerepository from 'fileRepository';
-import itemrepository from 'itemRepository';
-import useractionrepository from 'userActionRepository';
-import transfermanager from 'transferManager';
+﻿import filerepository from './sync/filerepository';
```

Comment:
shouldn't we eventually just drop all this `sync` nonsense?..

Reply:
Probably. Unless it is reused for downloads, but for now there are still references to it in jellyfin-web.

Ref: https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/site.js#L356-L369